app/src/main/java/vn/sun/music_68/ui/nowplaying/NowPlayingActivity.java

```diff
@@ -30,7 +30,7 @@ protected void initViews(Bundle savedInstanceState) {
         mFragments = new ArrayList<>();
         mFragments.add(FragmentMainPlaying.getInstance());
         mFragments.add(FragmentPlayingList.getInstance());
-        PlayPagerAdapter adapter = new PlayPagerAdapter(this, getSupportFragmentManager());
+        PlayPagerAdapter adapter = new PlayPagerAdapter( getSupportFragmentManager());
```

Comment:
Ctrl + Alt+L mỗi lần gõ xong chứ 
```suggestion
        PlayPagerAdapter adapter = new PlayPagerAdapter(getSupportFragmentManager());
```